src/app/components/MediaLoader/utils/buildSettings.ts

```diff
@@ -30,23 +30,14 @@ const buildSettings = ({
   adsEnabled = false,
   showAdsBasedOnLocation = false,
   embedded,
-  pathname,
 }: BuildConfigProps) => {
-  const embedUrl = pathname
-    ? getEmbedURL({
-        type: 'avEmbed',
-        mediaId: pathname,
-        queryString: '',
-      })
-    : '';
   // Base configuration that all media players should have
   const basePlayerConfig: PlayerConfig = {
     autoplay: true,
     product: 'news',
     enableToucan: true,
     appType: isAmp ? 'amp' : 'responsive',
     appName: service !== 'news' ? `news-${service}` : 'news',
-    externalEmbedUrl: embedUrl,
```

Comment:
Does this now mean that each page type will need to set `externalEmbedUrl` (the reason it was added here was so that we could have a default value for on demand TV and the other media page types)

Reply:
Yes having looked at it again each config will need to generate their own externalEmbedUrl. But this is only for page media that can be embedded offsite, which is only Article assets currently. Live pages can set this value as well because TIPO content will likely want to be embeddable offsite, but I'm not sure of OnDemand stuff.

Reply:
Ah in fact, you're right - I think we confirmed that it was not even possible to set embedding = allowed in the backend system for the on demand TV / radio, therefore embedding would not be supported. And because we don't support amp anymore, there's no need for an amp embed URL. Thanks for the clarification!